PandasWorkshop.ipynb

```diff
@@ -21,7 +21,7 @@
   },
```

Comment:
<p>It doesn't have to be here, but at some point we should explain the distinction between a Series and a DataFrame and tell them that calling <code>df['column_name']</code> selects a particular Series from a DataFrame. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/orioncohen/metal-bands-by-nation/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>